ixmp/core.py

```diff
@@ -271,7 +271,21 @@ def add_region_synonym(self, region, mapped_to):
         self._backend.set_node(region, synonym=mapped_to)
 
     def timeslices(self):
-        """Return all sub-annual time slices.
+        """Return all subannual timeslices defined in this Platform instance.
+
+        Timeslices are a way to represent subannual temporal resolution in
+        timeseries data. A timeslice consists of a **name** (e.g., 'january',
+        'summer'), a **category** (e.g., 'months', 'seasons'), and a
+        **duration** given relative to a full year.
+
+        The category and duration do not have any functional relevance within
```

Comment:
Clear _negative_ statements like this are super helpful :+1:

Along the same lines, do the durations within a category need to sum to 1? Or does the code not care?

Reply:
Code doesn't care...